libraries/engage/components/SheetList/components/Item/index.jsx

```diff
@@ -66,8 +70,10 @@ class Item extends Component {
       titleStyles += ` ${styles.disabled}`;
     }
 
+    const ref = isNested ? null : forwardedRef;
```

Comment:
Yes, because the `ref` is supposed to be passed down to the outermost element which is rendered by the SheetList Item. There are three situations where `this.renderContent()`is called. In two of those, the returned markup is wrapped by `Glow` or `div`. Only when it renders alone, the ref needs to be applied to the markup.